css-shapes-2/Overview.bs

```diff
@@ -351,7 +351,11 @@ Interpolating the ''shape()'' Function</h5>
 
 	''shape()'' and ''path()'' functions can be <l spec=css-values-4>[=interpolated=]</l> with each other
 	if their associated list of path data commands is the same length
-	and has the same commands, in order.
+	and has the same commands, in order, with the first value of the ''path()'' function interpolating with the
```

Comment:
perhaps s/value/command/, as I could naively interpret the first “value” being `m` or `M`?

Reply:
Done